runners/flink/flink_runner.gradle

```diff
@@ -309,6 +311,8 @@ def createValidatesRunnerTask(Map m) {
 
         // Flink reshuffle override does not preserve all metadata
         excludeTestsMatching 'org.apache.beam.sdk.transforms.ReshuffleTest.testReshufflePreservesMetadata'
+        // Flink redistribute override does not preserve all metadata
```

Comment:
This seems bad. Is it not possible to fix this (e.g. by reifying the metadata if needed)? I see that reshuffle didn't, but ideally the new, improved transform should do things the right way.

Reply:
OK I re-added and it succeeded. I think I added this during debugging. The non-portable runner works, if I have invoked the tests right. I'll watch the CI results.